src/status_im/accounts/handlers.cljs

```diff
@@ -129,11 +129,13 @@
   (let [accounts (->> (accounts-store/get-all)
                       (map (fn [{:keys [address] :as account}]
                              [address account]))
-                      (into {}))]
+                      (into {}))
+        view (if (empty? accounts)
```

Comment:
we can't add default view in the app-db, because we decide here which view will be first, so on the first launch it will be :chat